apps/nova/src/app/api/auth/problems/route.ts

```
@@ -0,0 +1,55 @@
+import { createClient } from '@tuturuuu/supabase/next/server';
+import { NextResponse } from 'next/server';
+
+interface Params {
+  params: Promise<{}>;
+}
+
+export async function GET(req: Request, {}: Params) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)